src/main/groovy/jenkins/automation/utils/CommonUtils.groovy

```diff
@@ -19,4 +24,31 @@ class CommonUtils {
             }
         }
     }
+
+/**
+ * Utility function to add extended email
+ *
+ * @param triggers List<String> triggers E.g Failure, Fixed etc...
+ * @param String emails Comma separated string of emails
+ * @param sendToDevelopers Default false,
+ * @param includeCulprits Default false,
+ * @param sendToRecipientList Default true
+ *
+ * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#common-utils" target="_blank">BDD job Example</a>
+ */
+
+    static void extendedEmail(context, List<String> triggers = ["Failure", "Fixed"], String emails, sendToDevelopers = false, includeCulprits = false, sendToRecipientList = true) {
```

Comment:
Should emails be a `List<String>` instead, to retain compatibility with the Builders? I don't have a strong opinion, but it seems to me that we should remain consistent